lib/cloudwatchlogs/client.go

```diff
@@ -109,47 +110,59 @@ func openAwsClient() (client *cloudwatchlogs.CloudWatchLogs, err error) {
 	return
 }
 
-func createGroupAndStream(client *cloudwatchlogs.CloudWatchLogs, group string, stream string) (token string, err error) {
+func createGroupAndStream(client *cloudwatchlogs.CloudWatchLogs, group string, stream string) (string, error) {
```

Comment:
The use of a named return here was communicating what the returned string was. The reader now has to reader through the end of the function to figure out what this value is.
I'd say we should keep the named return as a way to document the function, or add documentation explaining what the returned string is.

Reply:
There has been some ongoing debate among team members in the #golang channel about whether named or anonymous return arguments are better -- or, more specifically, whether to be explicit in `return` statements.   The weight of the opinion seemed to be towards explicit `return`s -- which suggests, but does not require, the use of anonymous return arguments in the function declaration.

Reply:
I think a happy middle ground here is to simply add a function comment indicating this method returns a token

Reply:
If named returns were a mistake they wouldn't be in the language. But you pick your style here, all I care about is that we keep expressing what this string is.

Reply:
I put the named return parameters back, but I'm also doing explicit `return`s in the code now.